qln/lndb.go

```diff
@@ -88,8 +88,10 @@ type LitNode struct {
 	// Nodes don't have Params; their SubWallets do
 	// Param *chaincfg.Params // network parameters (testnet3, segnet, etc)
 
-	// queue for async messages to RPC user
-	UserMessageBox chan string
+	// websocket connections of users to a message channel
+	UserWsCons map[int]chan lnutil.ChatMsg
```

Comment:
I could change the name to `UserChatChans` because like UserMessageBox it doesn't actually contain any websockets. It has to be in qln instead of litrpc so that qln/msghandler.go has access to it.